### PR TITLE
Feature/arm experimental

### PR DESCRIPTION
**Pull request by Maleware**
First draft of an small article about early access to our arm64 images

### CODE REVIEW

File: modules/concepts/pages/experimental-arm64-support.adoc [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+==== 2. Checkout arm64 demo branch ( release 23.11.0 )
+```bash
+git pull && git checkout spike/demos-on-arm
+```
+
+==== 3. Install your demo with stackablectl (e.g. nifi-kafka-druid-earthquake-data)
+```bash
+stackablectl demo install nifi-kafka-druid-earthquake-data --skip-release -s stacks/stacks-v2.yaml -d demos/demos-v2.yaml
+```
```
Comment (fhennig) [inline modules/concepts/pages/experimental-arm64-support.adoc:78]:
```suggestion
[source,bash]
----
stackablectl demo install nifi-kafka-druid-earthquake-data --skip-release -s stacks/stacks-v2.yaml -d demos/demos-v2.yaml
----
```

This is how you do code blocks in asciidoc ;)

Comment (Maleware) [reply modules/concepts/pages/experimental-arm64-support.adoc:74]: Thanks! Haven't seen your review till now. Appreciated!

File: modules/concepts/pages/experimental-arm64-support.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+[source,yaml]
+----
+spec:
+  image:
+    custom: docker.stackable.tech/stackable-experimental/kafka:3.5.1-stackable23.11.0
+    productVersion: 3.5.1
+----
+Once again, the key to success is to exchange `stackable` with `stackable-experimental` and point to the release 23.11.0.
+
+NOTE: Only 23.11.0 is working at this moment.
```
[the remark this answers is not in the document]
Comment (Maleware) [reply modules/concepts/pages/experimental-arm64-support.adoc:46]: Fine :)

File: modules/concepts/pages/experimental-arm64-support.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
+= Multi-Architecture for SDP
+:page-aliases: arm_support.adoc
```
Comment (fhennig) [inline modules/concepts/pages/experimental-arm64-support.adoc:2]:
```suggestion
```

Don't add this page alias, it's only for legacy pages.

**Review by adwk67 — DISMISSED** [review]
lgtm!

**Review by fhennig — CHANGES_REQUESTED** [review]
I'd suggest to rename the file to "arm64-support.adoc" (so no experimental) but add the experimental to the title (i.e. "Multi-architecture for SDP (experimental").

Otherwise if we stabilize this feature, the link will change, and that will be annoying.

**Review by fhennig — APPROVED** [review]
perfect, thanks!

File: modules/concepts/pages/arm64-support.adoc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    productVersion: 3.5.1
+----
+
+NOTE: Only 23.11.0 is working at this moment.
+
+=== Prepared Demos
+
+It is possible to run SDP demos on ARM64 architecture, but due to the changes that are necessary we have provided a separate branch to make this easier. Please follow the instructions below to run a demo on ARM.
+
+You will need access to a Kubernetes Cluster and to have the stackablectl https://docs.stackable.tech/home/stable/quickstart[command-line tool] installed.
```
Comment (lfrancke) [inline modules/concepts/pages/arm64-support.adoc:50]: Is stackablectl available as an ARM version?

File: modules/concepts/pages/arm64-support.adoc [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+:description: This page describes how to access ARM64 based SDP early
+:keywords: Multi-Architecture, infrastructure, docker, image, tags, early-access
+
+WARNING: This is an experimental feature of the Stackable Data Platform and still under https://github.com/stackabletech/issues/issues/463[development].
+
+== Early-Access
+
+SDP release 23.11.0 has been ported to the ARM64 architecture. These images are stored within the `stackable-experimental` organization in our https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable-experimental[Nexus]. 
+
+As they are not available by default so you need to first switch to the `stackable-experimental` organization before pulling them for use with the data platform.
```
[the remark this answers is not in the document]
Comment (Maleware) [reply modules/concepts/pages/arm64-support.adoc:11]: Through our default access ways. Default refers to `docker.stackable.tech/stackable/...`.  

I understand you need to somehow know what I mean. Is this more clear? :

switch from `docker.stackable.tech/stackable` to `docker.stackable.tech/stackable-experimental` before pulling images from the repository.

File: modules/concepts/pages/arm64-support.adoc (hunk 4 above)
Comment (Maleware) [reply modules/concepts/pages/arm64-support.adoc:50]: Yes https://docs.stackable.tech/management/stable/stackablectl/installation#_macos

**Comment by Maleware** [comment]
https://deploy-preview-558--stackable-docs.netlify.app/home/nightly/concepts/arm64-support current WIP

Currently deployed:
https://docs.stackable.tech/home/nightly/concepts/arm64-support

**Pull request merged**